tools/wptrunner/wptrunner/webdriver_server.py

```diff
@@ -239,7 +248,7 @@ def get_free_port():
             s.close()
 
 
-def wait_for_service(addr, timeout=15):
+def wait_for_service(addr, timeout=60):
```

Comment:
We need a longer timeout here since we need to wait for an iOS Simulator to boot up and to then launch the test app that implements CWTChromeDriver (our WebDriver implementation).